src/components/Search/SearchPageHeader.tsx

```diff
@@ -143,9 +144,17 @@ function SearchPageHeader({queryJSON}: SearchPageHeaderProps) {
             !isOffline &&
             !isAnyTransactionOnHold &&
             (selectedReports.length
-                ? selectedReports.every((report) => report.action === CONST.SEARCH.ACTION_TYPES.PAY && report.policyID && lastPaymentMethods[report.policyID])
+                ? selectedReports.every(
+                      (report) =>
+                          report.action === CONST.SEARCH.ACTION_TYPES.PAY &&
+                          report.policyID &&
+                          (!!lastPaymentMethods[report.policyID] || !!(lastPaymentMethods[report.policyID] as LastPaymentMethodType)?.DEFAULT),
```

Comment:
```suggestion
                          (!!lastPaymentMethods[report.policyID] && typeof lastPaymentMethods[report.policyID] === 'string' || !!(lastPaymentMethods[report.policyID] as LastPaymentMethodType)?.DEFAULT),
```

I think we need a check like this

Reply:
Good catch!